code/game/objects/items/devices/organ_module/passive/armor.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/organ_module/armor
 	name = "subdermal armor"
-	desc = "A set of subdermal steel plates, designed to provide additional impact protection while remaining lightweight."
+	desc = "A set of subdermal steel plates, designed to provide additional impact protection to the torso while remaining lightweight."
```

Comment:
It actually works by reducing all brute damage to THE WHOLE BODY, but it only works if it's installed in the torso.